libs/npm/core/src/components/table/interfaces/ITableHookOptions.ts

```diff
@@ -3,7 +3,11 @@ import { IdType, ITableInternalCell, ITableInternalColumn, ITableInternalRow } f
 export interface ITableHookOptions<T extends object> {
   onRowClick?: (row: ITableInternalRow<T>, event: React.MouseEvent) => void;
   onColumnClick?: (column: ITableInternalColumn<T>, event: React.MouseEvent) => void;
-  onCellClick?: (cell: ITableInternalCell<T>, event: React.MouseEvent) => void;
+  onCellClick?: (
```

Comment:
Including the row makes this function also work like clicking the row.  However, now we can ignore some columns being clicked on.